src/fft_2n.h

```diff
@@ -297,33 +304,51 @@ void Radix2<T>::fft(vec::Buffers<T>& output, vec::Buffers<T>& input)
 }
 
 /** Perform decimation-in-frequency FFT or inverse FFT
+ * Input buffer is in reversed-bit order. Hence butterfly operations can
```

Comment:
Skip a line before the lengthy description (that way Doxygen can format things properly)